### PR TITLE
Split: styling update

### PR DESCRIPTION
**Pull request by alexandrugu**
Made some small UI changes to match the UI on the create page.

### CODE REVIEW

File: src/pages/Split/components/Splitter.tsx [hunk 1]
```diff
@@ -192,14 +196,7 @@ export default function Splitter({ nft }: { nft: GovNft }) {
...
 
-            {vestingDuration !== 0 && (
-              <GovnftChart
-                startDate={selectedStartDate}
-                vestingDuration={vestingDuration}
-                cliffDuration={cliffDuration}
-              />
-            )}
-
+            {/* TODO: Hide button if vesting amount is > than available amount*/}
```
Comment (stas) [inline src/pages/Split/components/Splitter.tsx:199]: Can we remove this please.

**Review by stas — APPROVED** [review]
I believe this is good to go

cc @eyezick

**Comment by alexandrugu** [comment]
@eyezick there was one last conflict here, i've fixed. If you can take one final look, i think we might be ready to merge this one too.

**Review by eyezick — APPROVED** [review]
LGTM! ✅

**Pull request merged**